superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -125,4 +125,6 @@ RVF_HELP_ENABLED: False
 SUPERSET_SESSION_LIFETIME_SECONDS: "3600" # stale session timeout
 RVF_LOG_LEVEL: "INFO"
 RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60 * 60 * 24 }}"
-RVF_CACHE_TYPE: 'null'
\ No newline at end of file
+RVF_CACHE_TYPE: 'null'
+
+RVF_WALKME_ENABLED: True
```

Comment:
I think use of  RVF_WALKME_ENABLED  and WALKME_ENABLED  in superset-deployment.yaml  is missed

Reply:
Done